src/index.js

```diff
@@ -31,66 +31,68 @@ const detectNewETHAddress = async (str) => {
   }
 };
 
-const registerAddress = async (user, address) => {
-  const ogMembers = new Set(JSON.parse((await getAsync('ogMembers')) || '[]'));
-  ogMembers.add(address);
-  await Promise.all([
-    setAsync('ogMembers', JSON.stringify(Array.from(ogMembers))),
-    setAsync(address, user.id),
-    setAsync(`${user.id}|address`, address),
-  ]);
-};
+// const registerAddress = async (user, address) => {
```

Comment:
The function can be left alone and does not need to be commented out. We will likely use it again soon. Please uncomment.